nsv13/code/modules/overmap/boarding/boarding.dm

```diff
@@ -118,21 +163,23 @@
 			var/callsign = I
 			if(callsign <= 0)
 				callsign = "First Mate"
-				H.equipOutfit(/datum/outfit/pirate/space/boarding/lead) //review these
+				H.equipOutfit(/datum/outfit/pirate/space/boarding/lead)
 			else
 				callsign = "Crew"
 				var/list/pirate_kits = list(/datum/outfit/pirate/space/boarding/sapper, /datum/outfit/pirate/space/boarding/gunner) //review these
 				var/kit = pick(pirate_kits)
 				H.equipOutfit(kit)
-			var/beggings = strings(PIRATE_NAMES_FILE, "beginnings")
+			var/beginnings = strings(PIRATE_NAMES_FILE, "beginnings")
 			var/endings = strings(PIRATE_NAMES_FILE, "endings")
-			H.fully_replace_character_name(H.real_name, "[callsign] [pick(beggings)][pick(endings)]")
+			H.fully_replace_character_name(H.real_name, "[callsign] [pick(beginnings)][pick(endings)]")
 			H.mind.add_antag_datum(/datum/antagonist/pirate/boarder)
 			H.mind.assigned_role = "Pirate Boarder"
 			log_game("[key_name(H)] became a space pirate boarder.")
 			message_admins("[ADMIN_LOOKUPFLW(H)] became a space pirate boarder.")
 			operatives += H
-		 //No audio warning?
+		relay('nsv13/sound/effects/ship/boarding_pod.ogg', "<span class='userdanger'>You can hear several tethers attaching to the ship.</span>")
```

Comment:
Space Pirate Boarders having no audio warning is intentional.

Reply:
👍